apps/coscrad-frontend/src/store/index.ts

```diff
@@ -30,10 +31,12 @@ export const rootReducer = combineReducers({
     [TERMS]: termReducer,
     [PHOTOGRAPHS]: photographReducer,
     [TRANSCRIBED_AUDIO]: transcribedAudioReducer,
+    // For consistency, consider pluralizing the following identifiers (constants from each slice)
```

Comment:
Do this!

Reply:
done. Small gotcha- all the selectors had to be updated because they weren't using the constants to refer to the props on state.